### ISSUE TITLE
IntraDay query throws exception

### ISSUE THREAD

**Issue by ntrous** [AvapiDotNet/Avapi#3]
The below query throws a Object Reference Not Set exception when using anything other than n_1min

```
IAvapiResponse_TIME_SERIES_INTRADAY timeSeriesIntradayResponse =
                timeSeriesIntraday.Query(
                    "MSFT",
                    Const_TIME_SERIES_INTRADAY.TIME_SERIES_INTRADAY_interval.n_15min,
                    Const_TIME_SERIES_INTRADAY.TIME_SERIES_INTRADAY_outputsize.compact,
                    Const_TIME_SERIES_INTRADAY.TIME_SERIES_INTRADAY_datatype.json);
```

**Comment by sgiulians** [comment]
Found the error! We are going to provide the fix in the next couple of days. Thanks!

**Comment by sgiulians** [comment]
Hi ntrous,
FYI: We have just provided a new version that should fix the problem you raised (1.3.2)
Simo
